Android/src/org/droidplanner/android/fragments/widget/telemetry/MiniWidgetAttitudeSpeedInfo.kt

```diff
@@ -58,6 +58,8 @@ public class MiniWidgetAttitudeSpeedInfo : TowerWidget() {
 
     private var headingModeFPV: Boolean = false
 
+    private val MIN_VERTICAL_SPEED = 0.10
```

Comment:
@guiseco Looks good! Can you add a comment or update the constant's name to indicate the unit for clarity.